systemProperties.adoc

```diff
@@ -0,0 +1,25 @@
+== System Properties
+Below, a number of relevant System Properties are listed.
+
+
+NOTE: All boolean properties below are read using link:https://javadoc.io/static/net.openhft/chronicle-core/2.23ea13/net/openhft/chronicle/core/Jvm.html#getBoolean-java.lang.String-[net.openhft.chronicle.core.Jvm.getBoolean(java.lang.String)], and so are enabled if either `-Dflag` or `-Dflag=true` or `-Dflag=yes`
+
+.System properties
+[cols=4*, options="header"]
+|===
+| Property Key | Default | Description | Java Variable Name (Type)
+| chronicle.network.callOnMissedHeartbeatOnDisconnect |`false` | If enabled, calls HeartBeatListener when we do not receive a heartbeat | _CALL_MISSED_HEARTBEAT_ON_DISCONNECT_ (boolean)
+| client.timeout | 500 | Number of milliseconds after which a client is timed out if they cannot to a host port | _pausePeriodMs_ (int)
+| connectionStrategy.pauseMillisBeforeReconnect | 500 | Allows control of a backoff strategy, and returns how long in milliseconds to pause before attempting reconnect | pauseMillisBeforeReconnect (long)
+| connectionStrategy.pause.max.secs | 5 | Controls how frequently the ConnectionStrategy.connect method is called | _maxPauseSec_ (long)
+| connectionStrategy.pause.min.secs | 5 | Controls how frequently the ConnectionStrategy.connect method is called | _minPauseSec_ (long)
+| connectionStrategy.socketConnectionTimeoutMs | 1 | Minimum time before message displayed: "Timed out attempting to connect to " + socketAddress | socketConnectionTimeoutMs (int)
+| disable.tcp.nodelay | `false` | If enabled, displays message 'tcpNoDelay disabled' and allows sending of successive small packets on the network | _DISABLE_TCP_NODELAY_ (boolean)
+| ssl.test.payload | `false` | If enabled, triggers testDataConnection to encode ByteBuffer message while there is at least one remaining element in the ByteBuffer | _SEND_DATA_BEFORE_SSL_HANDSHAKE_ (boolean)
+| tcp.client.buffer.size | 0 | Gets or sets the tcp buffer size | _tcpBufferSize_ (int)
```

Comment:
This actually defaults to 4 MiB (kind of, it's a long story)

Reply:
it sets the value for `SO_RCVBUF` (see https://docs.oracle.com/javase/7/docs/api/java/net/ServerSocket.html#setReceiveBufferSize(int))